src/components/gallery/gallery-modal.hbs

```diff
@@ -4,7 +4,7 @@
       <button class="modal__close" data-modal-hide="gallery-modal">
         <span class="sr-only">close</span>
         <svg class="icon icon-close">
-          <use xlink:href="../sprite.svg#close"></use>
+          <use xlink:href="{{rootPath}}/sprite.svg#close"></use>
```

Comment:
@brianelton does this work on its own like this? I don't see the `rootPath` property being passed into the galleryModal init call anywhere.

Reply:
I undid that for now. Have to understand how to put it in and will return to it.